CONTRIBUTING.md

```diff
@@ -52,31 +52,26 @@ In addition to running npm install in the root of the project (to install the de
 # At project root
 npm install
 ```
+To test both samples you will need the following configured in your developer org:
 
-Then you need to setup the following environment variables
-
-```bash
-export CLIENT_ID={{yourAppClientId}}
-export OKTA_DOMAIN={{yourOktaOrgDomain}}
-```
-
-> **NOTE:** Use only the domain part of your org url while setting OKTA_DOMAIN environment.
+* A SPA application
+* A test user account with a known username and password. Note that the USERNAME should be of the form "username@email.com"
 
-* E.g - If your org url is https://myorg.oktapreview.okta.com, your OKTA_DOMAIN should be myorg.oktapreview
+Then you need to setup the following environment variables
```

Comment:
I think we can remove this line - as its purpose is repeated in the following sentence.

Reply:
Done